local/postman/puris-integration-test.postman_collection.json

```diff
@@ -1,10 +1,10 @@
 {
 	"info": {
-		"_postman_id": "8d1f2083-27df-4f47-8f1f-16ab64224be5",
+		"_postman_id": "366237ef-086a-48bb-b723-c0da31f8e1ca",
 		"name": "puris-integration-test",
-		"description": "The collection assumes the following roles:\n\n- Customer PURIS (upper tier)\n    \n- Supplier PURIS (lower tier)\n    \n\nIt creates the opposite partner and the respective view on the semiconductor material or product.\n\nTo run the tests correctly, take the environment and fill the respective secrets:\n\n- CUSTOMER_EDC_API_KEY\n- CUSTOMER_PURIS_BACKEND_API_KEY\n- SUPPLIER_EDC_API_KEY\n- SUPPLIER_PURIS_BACKEND_API_KEY\n    \n\nWhen using the another environment, update / create a new environment.",
+		"description": "The collection assumes the following roles:\n\n- Customer PURIS (upper tier)\n- Supplier PURIS (lower tier)\n    \n\nIt creates the opposite partner and the respective view on the semiconductor material or product.\n\nTo run the tests correctly, take the environment and fill the respective secrets:\n\n- CUSTOMER_EDC_API_KEY\n- CUSTOMER_PURIS_BACKEND_API_KEY\n- CUSTOMER__MANAGE_CLIENT_SECRET\n- SUPPLIER_EDC_API_KEY\n- SUPPLIER_MANAGE_CLIENT_SECRET\n    \n\nWhen using the another environment, update / create a new environment.",
```

Comment:
Does it really make sense that supplier backend api key was removed from this list?

Reply:
No, you're right. Fixed!